### ISSUE TITLE
XML double parsing

### ISSUE THREAD

**Issue by alessandrosettimi** [ADVRHumanoids/XBotGUI#30]
Find out why in some PCs double are correctly parsed with dot separator and in other PCs the comma separator is needed.

**Comment by alessandrosettimi** [comment]
In c0d59cca0e9c7a03ae1aa00b13399ee0c7a39405 a cmake flag to switch between commas and dots as decimal separator has been added. The idea is to modify it automatically in the cmake when the cause for this different ways of parsing is found.

**Comment by alessandrosettimi** [comment]
Seems that with 7e91fbe4cfd212b87ff3581e5c8aec34e1dcb191 I forced the dot as decimal separator. See [this](https://stackoverflow.com/questions/12316972/how-to-tweak-stdstod-string-to-double-for-decimal-separator-and-number-of-di) for more details. In e86fed9595531d66af84ff0362389362bd0ffeef I changed the default for **USING_COMMA_AS_DECIMAL_SEPARATOR** as **false**. So now the parsing should work in every PC, the ccmake flag however can be used to toggle the behavior. I close this.
